dynawo/sources/Modeler/Common/DYNSubModel.cpp

```diff
@@ -1395,7 +1395,7 @@ SubModel::printInitValuesVariables(std::ofstream& fstream) {
   if (calculatedVarsInit_.size() > 0) {
     fstream << " ====== INIT CALCULATED VARIABLES VALUES ======\n";
     const vector<string>& calculatedVarNames = (*this).getCalculatedVarNamesInit();
-    for (unsigned int i = 0, iEnd = calculatedVarsInit_.size(); i < iEnd; ++i)
+    for (size_t i = 0, iEnd = calculatedVarsInit_.size(); i < iEnd; ++i)
```

Comment:
Not related

Reply:
do you mean with clang?

Reply:
Yes, with clang++ I have this error if I don't fix this line : 

error: implicit conversion loses integer precision: 'std::vector::size_type' (aka 'unsigned long') to 'unsigned int' [-Werror,-Wshorten-64-to-32]